main.html

```diff
@@ -1,47 +1,72 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-<meta charset="UTF-8">
-<title>Page Title</title>
-<link rel="stylesheet" href="main.css">
-<link rel="stylesheet" href="normalize.css">
+    <meta charset="UTF-8">
+    <meta content='width=device-width, initial-scale=1.0, user-scalable=0' name='viewport'/>
+    <title>Page Title</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.css">
+    <link rel="stylesheet" href="main.css">
 </head>
 <body>
     <header>
         <div class="header">
             <figure class="header__image">
-                <img src = "lyft_logo.png" alt=""/>
+                <img src="lyft_logo.png" class ="header__image-source" alt="lyft-logo"/>
             </figure>
             <div class="header__small-container">
                 <div class="header__search">
-                    <textarea rows="3" cols="1" placeholder="Where are you go" class="header__textarea"></textarea>
+                    <textarea rows="3" cols="1" placeholder="Where are you going?" class="header__textarea"></textarea>
                     <div class="header__glass">
                         <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="header__glass-source" viewBox="0 0 16 16">
                             <path d="M11.742 10.344a6.5 6.5 0 1 0-1.397 1.398h-.001c.03.04.062.078.098.115l3.85 3.85a1 1 0 0 0 1.415-1.414l-3.85-3.85a1.007 1.007 0 0 0-.115-.1zM12 6.5a5.5 5.5 0 1 1-11 0 5.5 5.5 0 0 1 11 0z"/>
-                          </svg>
+                        </svg>
+                    </div>
+                    <div class="header__point">
+                        <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="header__point-img" viewBox="0 0 16 16">
+                            <path d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0zm-8 3a3 3 0 1 0 0-6 3 3 0 0 0 0 6z"/>
+                        </svg>
                     </div>
                 </div>
                 <div class="header__burger">
-                    <img src="burger.png" alt=""/>
+                    <img src="burger.png" class="header__burger-img" alt="burger-button"/>
                 </div>
+                <nav class="header__nav">
+                    <div class="header__list">
```

Comment:
do we really need this `header__list`? can we collapse them (all styles) into one tag?

Reply:
+